.github/workflows/compilation_on_android_ubuntu.yml

```diff
@@ -475,7 +477,7 @@ jobs:
           sudo apt-get update &&
           sudo apt install -y g++-multilib lib32gcc-9-dev
 
-      - name: run spec tests x86_32
-        if: env.TEST_ON_X86_32 == 'true'
+      - name: run tests x86_32
+        if: env.TEST_ON_X86_32 == 'true' && matrix.test_option != '$WASI_TEST_OPTIONS'
```

Comment:
What's the reason for not running wasi tests on x86-32?

Reply:
I think it'd be worth running those tests on x86-32 as well (to make sure WASI implementation works on those platforms as well).

Reply:
Do we want to run wasi tests on x86-32? For the output of the CI, I found that x86-32 wasi tests are ignored:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/3834321537/jobs/6526922968
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/3834321537/jobs/6526923240

Seems that we need to change the below lines in the CI file
```yml
- name: set env variable(if x86_32 test needed)
        if: >
          (matrix.test_option == '$DEFAULT_TEST_OPTIONS' || matrix.test_option == '$THREADS_TEST_OPTIONS')
          && matrix.running_mode != 'fast-jit' && matrix.running_mode != 'jit'
        run: echo "TEST_ON_X86_32=true" >> $GITHUB_ENV
```
into:
```yml
- name: set env variable(if x86_32 test needed)
        if: >
          (matrix.test_option == '$DEFAULT_TEST_OPTIONS' || matrix.test_option == '$THREADS_TEST_OPTIONS'
           || matrix.test_option == '$WASI_TEST_OPTIONS')
          && matrix.running_mode != 'fast-jit' && matrix.running_mode != 'jit'
        run: echo "TEST_ON_X86_32=true" >> $GITHUB_ENV
```